docs/assemblies/hotfixing-the-data-plane.adoc

```diff
@@ -0,0 +1,21 @@
+[id="assembly_hotfixing-the-data-plane"]
+= Hotfixing the data plane
+
+:context: dataplane
+
+[role="_abstract"]
+The OpenStack DataPlane can be updated when hotfix content is available. Hotfix content
+can be delivered as RPM packages or container images. How the software is
+installed on the data plane nodes determines how the hotfix content needs to be
+applied.
+
+include::proc_hotfixing-the-data-plane-rpm-content.adoc[leveloffset=+1]
```

Comment:
This include needs to be moved to after all the text in the assembly. Can you move it to just before the  include on line 19?

Reply:
Updated with suggestion